eng/common/scripts/Generate-PR-Diff.ps1

```diff
@@ -28,7 +28,7 @@ function Get-ChangedServices
     [string[]] $ChangedFiles
   )
 
-  $changedServices = $ChangedFiles | Foreach-Object { if ($_ -match "sdk/([^/]+)") { $matches[1] } } | Sort-Object -Unique
+  [string[]] $changedServices = $ChangedFiles | Foreach-Object { if ($_ -match "sdk/([^/]+)") { $matches[1] } } | Sort-Object -Unique
```

Comment:
You can also probably wrap in `@(...)` if your goal is to aways have an array.

Reply:
You're right! We just need to ensure this is an array type.

The part that is doing the actual lifting us the `,` in front of the return.